CONTRIBUTING.md

```diff
@@ -190,6 +224,12 @@ running:
 $ go test -v cloud.google.com/go/...
```

Comment:
@tbpg should this also be changed to explicitly target the current development directory?

Reply:
Yes. It might also be good to note that this doesn't run the tests in other modules. So, this won't run the tests for the `datastore` module.